packages/platform/platform-fastify/src/components/PlatformFastify.ts

```
@@ -0,0 +1,337 @@
+          }
+        });
+      } catch (er) {
+        logger().warn({
+          error_name: er.code,
+          error_message: er.message
+        });
+      }
+    });
+  }
+
+  mapHandler(handler: Function, metadata: PlatformHandlerMetadata) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)